README.md

```diff
@@ -1,5 +1,8 @@
 Python container images
-====================
+=======================
+s2i-python-container 2.7 Quay.io status: [![Docker Repository on Quay](https://quay.io/repository/centos7/python-27-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/python-27-centos7)
+
+s2i-python-container 3.8 Quay.io status: [![Docker Repository on Quay](https://quay.io/repository/centos7/python-38-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/python-38-centos7)
```

Comment:
One last thing, could we make this on one line?

Reply:
s2i-python-container 2.7 [status], s2i-python-container 3.8 [status]?

Reply:
Yes, is it possible?

Reply:
Sure